### PR TITLE
Default cover page

### PR DESCRIPTION
**Pull request by satvikmaker**
Added Default cover page for the books

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
    <script id="book-template" type="text/x-handlebars-template">
    {{#each this}}
      <div class="book">
-       <img src="http://covers.openlibrary.org/b/isbn/{{isbn}}-M.jpg" alt={{name}} class="cover">
+       <img src="http://covers.openlibrary.org/b/isbn/{{isbn}}-M.jpg" alt="default.jpg" class="cover">
```
Comment (hemanth) [inline index.html]: `alt` doesn't work as your expecting it to work, the `alt` attribute is for alternative text.

Comment (satvikmaker) [inline index.html]: OMG! What was I thinking! O.o That commit was not delivered by me -_-

**Comment by satvikmaker** [comment]
Please have a look at the current commit.

**Comment by satvikmaker** [comment]
Not sure if its the Pizza effect! Something is really wrong with me... Please verify the code. I closed by mistake.

**Comment by hemanth** [comment]
Heh heh, thanks!

**Pull request merged**